tests/cross_functional/system_test/test_mcg_replication_with_disruptions.py

```diff
@@ -402,3 +417,351 @@ def test_log_based_replication_with_disruptions(
             object_amount=5,
         )
         logger.info("No issues seen with the MCG bg feature validation")
+
+
+@mcg
+@magenta_squad
+@system_test
+class TestMCGReplicationWithVersioningSystemTest:
```

Comment:
Log-based replication assumes connectivity to AWS:
```suggestion
@skipif_aws_creds_are_missing
@skipif_disconnected_cluster
class TestMCGReplicationWithVersioningSystemTest:
```